x-pack/plugins/watcher/public/models/index.d.ts

```diff
@@ -3,7 +3,6 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
-export const PLUGIN = {
-  ID: 'watcher'
-};
+declare module 'plugins/watcher/models/watch' {
```

Comment:
Do we have to manually create this file and maintain it? Normally it is automatically generated by Typescript from our exposed `index.ts` file.

Reply:
I am also curious about this

Reply:
This module is not in TS, so yes I had to create it manually.

Reply:
@bmcconaghy For missing module typings it would be better to use the `/x-pack/typings` folder then.